Library/Formula/v8.rb

```diff
@@ -21,6 +21,7 @@ class V8 < Formula
 
   depends_on :python => :build # gyp doesn't run under 2.6 or lower
   depends_on "readline" => :optional
+  depends_on "icu4c" => :recommended
```

Comment:
I'm probably in favour of this too. The whole `icu4c` package is not tiny and not everyone on OS X needs or wants it.

Otherwise, this PR looks great.